### ISSUE TITLE
Reevaluate unaliased messages - Issues

### ISSUE THREAD

**Issue by pStyl3** [loot/skyrim#454]
Please see the main issue for more information.

---
Possible message anchors that could be used to refactor these messages:

```yaml
  - &corrupt
    type: warn
    content:
      - lang: en
        text: 'This file is corrupt and should not be used.'

  - &obsolete
    type: warn
    content:
      - lang: en
        text: 'Obsolete. Update to the latest version. %1%'
```

---

# Plugins reevaluated:
- [x] Unofficial Skyrim Patch.esm
- [x] Witcher2-v3.0 Aedirn-Whiterun.esp
- [x] Witcher2-v3.0 La Valette-Reach.esp
- [x] Witcher2-v3.0 Nilfgaard-Pale.esp
- [x] Witcher2-v3.0 Redania-Haarfingar.esp
- [x] Witcher2-v3.0 Temeria-Eastmarsh.esp
- [x] Witcher2-v3.0 Ulfric-Roche.esp
- [x] Witcher2-v3.0 Viperassassin-DB.esp
- [x] increaseddifficulty.esp
- [x] WoodChoppingExtended.esp
- [x] CK''s Improved Windhelm.esp
- [x] SoS - The Dungeons.esp
- [x] SoS - The Wilds.esp
- [x] SoS - Civilization.esp
- [x] Volrath - Dragonreach - Brazier fix( - 1\.1)\.esp
- [x] FastTravelAlle.esp
- [x] Fasttravelfromanywhere.esp
- [x] WTF.esp
- [x] [The Spice of Life.esm](https://www.nexusmods.com/skyrim/mods/16668/) **NSFW**
- [x] XFLMain.esm

```yaml
  - name: 'The Spice of Life.esm'
    msg:
      - type: warn
        condition: 'checksum("The Spice of Life.esm",A75F2F6D)'
        content:
          - lang: en
            text: 'File not created with Creation Kit.'
          - lang: de
            text: 'Die Datei wurde nicht mit dem Creation Kit hergestellt.'
          - lang: es
            text: 'Archivo no creado con el Creation Kit.'
          - lang: ja
            text: 'Creation Kitで作成されたファイルではありません。'
```

---

# Done:
<details>
  <summary>Replaced messages</summary>

```yaml
  - name: 'Witcher2-v3.0 Aedirn-Whiterun.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'Author states on download page that CTDs occur when player wears an armor.  He does not say which esp causes this.'
          - lang: de
            text: 'Der Autor gibt auf der Download-Seite an, dass CTDs auftreten, wenn der Spieler eine Rüstung trägt. Er sagt nicht, welche esp dies verursacht.'
          - lang: ja
            text: 'ダウンロードページにて製作者より、プレイヤーが防具を装備したときにCTDが発生すると言及されています。どのEspがCTDを引き起こすのかは述べられていません。'
          - lang: ru
            text: 'На странице загрузки автор утверждает, что броня может вызвать вылет игры, но не уточняет, какой esp вызывает его.'
      - *obsolete
    dirty:
      - <<: *dirtyPlugin
        crc: 0x6f9af605
        udr: 4

  - name: 'Witcher2-v3.0 La Valette-Reach.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'Author states on download page that CTDs occur when player wears an armor.  He does not say which esp causes this.'
          - lang: de
            text: 'Der Autor gibt auf der Download-Seite an, dass CTDs auftreten, wenn der Spieler eine Rüstung trägt. Er sagt nicht, welche esp dies verursacht.'
          - lang: ja
            text: 'ダウンロードページにて製作者より、プレイヤーが防具を装備したときにCTDが発生すると言及されています。どのEspがCTDを引き起こすのかは述べられていません。'
          - lang: ru
            text: 'На странице загрузки автор утверждает, что броня может вызвать вылет игры, но не уточняет, какой esp вызывает его.'

  - name: 'Witcher2-v3.0 Nilfgaard-Pale.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'Author states on download page that CTDs occur when player wears an armor.  He does not say which esp causes this.'
          - lang: de
            text: 'Der Autor gibt auf der Download-Seite an, dass CTDs auftreten, wenn der Spieler eine Rüstung trägt. Er sagt nicht, welche esp dies verursacht.'
          - lang: ja
            text: 'ダウンロードページにて製作者より、プレイヤーが防具を装備したときにCTDが発生すると言及されています。どのEspがCTDを引き起こすのかは述べられていません。'
          - lang: ru
            text: 'На странице загрузки автор утверждает, что броня может вызвать вылет игры, но не уточняет, какой esp вызывает его.'
    dirty:
      - <<: *dirtyPlugin
        crc: 0xb44cf9de
        udr: 1

  - name: 'Witcher2-v3.0 Redania-Haarfingar.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'Author states on download page that CTDs occur when player wears an armor.  He does not say which esp causes this.'
          - lang: de
            text: 'Der Autor gibt auf der Download-Seite an, dass CTDs auftreten, wenn der Spieler eine Rüstung trägt. Er sagt nicht, welche esp dies verursacht.'
          - lang: ja
            text: 'ダウンロードページにて製作者より、プレイヤーが防具を装備したときにCTDが発生すると言及されています。どのEspがCTDを引き起こすのかは述べられていません。'
          - lang: ru
            text: 'На странице загрузки автор утверждает, что броня может вызвать вылет игры, но не уточняет, какой esp вызывает его.'
      - *obsolete

  - name: 'Witcher2-v3.0 Temeria-Eastmarsh.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'Author states on download page that CTDs occur when player wears an armor.  He does not say which esp causes this.'
          - lang: de
            text: 'Der Autor gibt auf der Download-Seite an, dass CTDs auftreten, wenn der Spieler eine Rüstung trägt. Er sagt nicht, welche esp dies verursacht.'
          - lang: ja
            text: 'ダウンロードページにて製作者より、プレイヤーが防具を装備したときにCTDが発生すると言及されています。どのEspがCTDを引き起こすのかは述べられていません。'
          - lang: ru
            text: 'На странице загрузки автор утверждает, что броня может вызвать вылет игры, но не уточняет, какой esp вызывает его.'
      - *obsolete

  - name: 'Witcher2-v3.0 Ulfric-Roche.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'Author states on download page that CTDs occur when player wears an armor.  He does not say which esp causes this.'
          - lang: de
            text: 'Der Autor gibt auf der Download-Seite an, dass CTDs auftreten, wenn der Spieler eine Rüstung trägt. Er sagt nicht, welche esp dies verursacht.'
          - lang: ja
            text: 'ダウンロードページにて製作者より、プレイヤーが防具を装備したときにCTDが発生すると言及されています。どのEspがCTDを引き起こすのかは述べられていません。'
          - lang: ru
            text: 'На странице загрузки автор утверждает, что броня может вызвать вылет игры, но не уточняет, какой esp вызывает его.'

  - name: 'Witcher2-v3.0 Viperassassin-DB.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'Author states on download page that CTDs occur when player wears an armor.  He does not say which esp causes this.'
          - lang: de
            text: 'Der Autor gibt auf der Download-Seite an, dass CTDs auftreten, wenn der Spieler eine Rüstung trägt. Er sagt nicht, welche esp dies verursacht.'
          - lang: ja
            text: 'ダウンロードページにて製作者より、プレイヤーが防具を装備したときにCTDが発生すると言及されています。どのEspがCTDを引き起こすのかは述べられていません。'
          - lang: ru
            text: 'На странице загрузки автор утверждает, что броня может вызвать вылет игры, но не уточняет, какой esp вызывает его.'

  - name: 'Unofficial Skyrim Patch.esm'
    msg:
      - type: error
        content:
          - lang: en
            text: 'This esm file WILL damage your game. We strongly advise that you use the [Unofficial Skyrim Legendary Edition Patch](https://www.nexusmods.com/skyrim/mods/71214). Do not modify its extension.'
          - lang: de
            text: 'Diese esm Datei WIRD ihr Spiel beschädigen. Wir empfehlen mit Nachdruck, dass Sie den [Unofficial Skyrim Legendary Edition Patch](https://www.nexusmods.com/skyrim/mods/71214) benutzen. Modifizieren Sie nicht die Dateityp-Erweiterung.'

  - name: 'increaseddifficulty.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'This mod is not properly mastered to Skyrim.esm. It will not change or add anything in your game.'
          - lang: de
            text: 'Diese Mod ist nicht richtig für Skyrim.esm gemastert. Es wird nichts in Ihrem Spiel ändern oder hinzufügen.'
          - lang: ja
            text: 'このModはSkyrim.esmを正しくマスターファイルとして認識していません。変更内容をそのまま複製してしまいます。'
        condition: 'checksum("increaseddifficulty.esp",DFD6D81E)'

  - name: 'WoodChoppingExtended.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'This file contains 00000000 FormID errors and errors with two REFR records and should be removed.'
          - lang: de
            text: 'Diese Datei enthält 00000000 FormID-Fehler und Fehler mit zwei REFR-Datensätzen und sollte entfernt werden.'
          - lang: ja
            text: 'このファイルには00000000 FormIDエラーおよび2つのREFRレコードエラーが含まれています。エラーを取り除くべきでしょう。'
          - lang: ru
            text: 'Этот файл содержит ошибки 00000000 FormID, ошибки с двумя записями REFR и должен быть удален.'

  - name: 'CK''s Improved Windhelm.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'This plugin is reported to cause frequent crashes. Use with caution.'
          - lang: de
            text: 'Es wird berichtet, dass dieses Plugin häufige Abstürze verursacht. Mit Vorsicht zu verwenden.'
          - lang: ja
            text: 'このプラグインは頻繁にCTDを引き起こすとの報告があります。注意して使用してください。'
          - lang: ru
            text: 'Сообщалось, что этот плагин вызывал частые вылеты. Используйте с осторожностью.'

  - name: 'SoS - The Dungeons.esp'
    msg:
      - <<: *patchProvided
        subs: [ 'Climates Of Tamriel Dungeons Hardcore' ]
        condition: 'file("ClimatesOfTamriel-Dungeons-Hardcore.esp") and not file("SoS - The Dungeons-PatchCoTHardcore.esp")'
      - <<: *patchProvided
        subs: [ 'RLO Dungeons' ]
        condition: 'file("Realistic Lighting Overhaul - Dungeons.esp") and not file("SoS - The Dungeons-PatchRLODungeons.esp")'
      - <<: *patchProvided
        subs: [ 'RLO Dungeon Lighting' ]
        condition: 'file("Realistic Lighting Overhaul - Dungeon Lighting.esp") and not file("SoS - The Dungeons-PatchRLODungeonLighting.esp")'
      - <<: *patchProvided
        subs: [ '(RCRN AE) Realistic Colors & Real Nights AE' ]
        condition: 'file("rcrnShaders.esp") and not file("SoS - The Dungeons-PatchRCRN.esp")'
      - <<: *patchProvided
        subs: [ '(RCRN  Plus) Realistic Colors & Real Nights Plus' ]
        condition: 'active("rcrnPlus.esp") and not active("SoS - The Dungeons-PatchRCRN.esp")'
      - type: say
        content:
          - lang: en
            text: 'Do not use the Crash Fix ESP for this mod. It is flagged as a master and is only meant as a last resort for fixing crashes.'
          - lang: de
            text: 'Verwenden Sie für diesen Mod nicht die Crash Fix ESP. Es ist als Master gekennzeichnet und nur als letzter Ausweg zur Behebung von Abstürzen gedacht.'
          - lang: ja
            text: 'このMod用のCrash Fix ESPは使用しないでください。マスターファイルとしてのフラグが立ちます。クラッシュを修正するための最後の手段としてください。'
          - lang: ru
            text: 'Не используйте Crash Fix ESP для этого мода. Он помечен как мастерфайл и предназначен только как последнее средство для исправления вылетов.'

  - name: 'SoS - The Wilds.esp'
    msg:
      - <<: *patchProvided
        subs: [ 'Climates Of Tamriel' ]
        condition: 'file("ClimatesOfTamriel.esm") and not file("SoS - The Wilds-PatchCoT.*\.esp") and not file("WAO-WeatherAndAmbienceOverhaul.esp")'
      - <<: *patchProvided
        subs: [ '(RCRN AE) Realistic Colors & Real Nights AE' ]
        condition: 'file("rcrnShaders.esp") and not file("SoS - The Wilds-PatchRCRN.esp")'
      - <<: *patchProvided
        subs: [ '(RCRN  Plus) Realistic Colors & Real Nights Plus' ]
        condition: 'active("rcrnPlus.esp") and not active("SoS - The Wilds-PatchRCRN.esp")'
      - type: say
        content:
          - lang: en
            text: 'Do not use the Crash Fix ESP for this mod. It is flagged as a master and is only meant as a last resort for fixing crashes.'
          - lang: de
            text: 'Verwenden Sie für diesen Mod nicht die Crash Fix ESP. Es ist als Master gekennzeichnet und nur als letzter Ausweg zur Behebung von Abstürzen gedacht.'
          - lang: ja
            text: 'このMod用のCrash Fix ESPは使用しないでください。マスターファイルとしてのフラグが立ちます。クラッシュを修正するための最後の手段としてください。'
          - lang: ru
            text: 'Не используйте Crash Fix ESP для этого мода. Он помечен как мастерфайл и предназначен только как последнее средство для исправления вылетов.'
    dirty:
      - <<: *dirtyPlugin
        crc: 0xc1743294
        itm: 8

  - name: 'SoS - Civilization.esp'
    msg:
      - <<: *patchProvided
        subs: [ 'Climates Of Tamriel' ]
        condition: 'file("ClimatesOfTamriel.esm") and not file("SoS - Civilization(-PatchCoT| -PatchCoTSounds.esp|-PatchCoTWinterEdition| -PatchCoTSoundsWinterEdition)\.esp") and not file("WAO-WeatherAndAmbienceOverhaul.esp")'
      - <<: *patchProvided
        subs: [ 'RLO Major City Interiors' ]
        condition: 'file("Realistic Lighting Overhaul - Major City Interiors.esp") and not file("SoS - Civilization-PatchRLOMajorCityInteriors.esp")'
      - <<: *patchProvided
        subs: [ 'RLO Minor Cities and Town Interiors.esp' ]
        condition: 'file("Realistic Lighting Overhaul - Minor Cities and Town Interiors.esp") and not file("SoS - Civilization-PatchRLOMinorCities.esp")'
      - <<: *patchProvided
        subs: [ '(RCRN AE) Realistic Colors & Real Nights AE' ]
        condition: 'file("rcrnShaders.esp") and not file("SoS - Civilization-PatchRCRN.esp")'
      - <<: *patchProvided
        subs: [ '(RCRN  Plus) Realistic Colors & Real Nights Plus' ]
        condition: 'active("rcrnPlus.esp") and not active("SoS - Civilization-PatchRCRN.esp")'
      - type: say
        content:
          - lang: en
            text: 'Do not use the Crash Fix ESP for this mod. It is flagged as a master and is only meant as a last resort for fixing crashes.'
          - lang: de
            text: 'Verwenden Sie für diesen Mod nicht die Crash Fix ESP. Es ist als Master gekennzeichnet und nur als letzter Ausweg zur Behebung von Abstürzen gedacht.'
          - lang: ja
            text: 'このMod用のCrash Fix ESPは使用しないでください。マスターファイルとしてのフラグが立ちます。クラッシュを修正するための最後の手段としてください。'
          - lang: ru
            text: 'Не используйте Crash Fix ESP для этого мода. Он помечен как мастерфайл и предназначен только как последнее средство для исправления вылетов.'

  - name: 'Volrath - Dragonreach - Brazier fix( - 1\.1)\.esp'
    msg:
      - type: error
        content:
          - lang: en
            text: 'If using V1.1 [RCRN](http://www.nexusmods.com/skyrim/mods/1875) is required, otherwise your game will crash.'
          - lang: de
            text: 'Wenn Sie V1.1 verwenden, ist [RCRN](http://www.nexusmods.com/skyrim/mods/1875) erforderlich, ansonsten stürzt Ihr Spiel ab.'
          - lang: ja
            text: 'V1.1を使用している場合は[RCRN](http://www.nexusmods.com/skyrim/mods/1875)をインストールしないとゲームがクラッシュします。'
          - lang: ru
            text: 'Если используется версия 1.1, тогда требуется мод [RCRN](http://www.nexusmods.com/skyrim/mods/1875), иначе ваша игра может вылетать.'
        condition: 'not file("rcrnShaders.esp") and not file("RCRN (Classic|Legacy|Pure) - Hybrid Shaders\.esp")'

  - name: 'FastTravelAlle.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'See [mod page comments](http://www.nexusmods.com/skyrim/mods/27321) for error reports.'
          - lang: de
            text: 'Siehe [Kommentare zur Mod-Seite](http://www.nexusmods.com/skyrim/mods/27321) für Fehlerberichte.'
          - lang: ja
            text: '[公開ページのPOSTS欄](http://www.nexusmods.com/skyrim/mods/27321)にあるエラー報告に目を通しておいてください。'
          - lang: ru
            text: 'Смотрите [комментарии на странице мода](http://www.nexusmods.com/skyrim/mods/27321) для сообщения об ошибках.'

  - name: 'Fasttravelfromanywhere.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'See [mod page comments](http://www.nexusmods.com/skyrim/mods/27321) for error reports.'
          - lang: de
            text: 'Siehe [Kommentare zur Mod-Seite](http://www.nexusmods.com/skyrim/mods/27321) für Fehlerberichte.'
          - lang: ja
            text: '[公開ページのPOSTS欄](http://www.nexusmods.com/skyrim/mods/27321)にあるエラー報告に目を通しておいてください。'
          - lang: ru
            text: 'Смотрите [комментарии на странице мода](http://www.nexusmods.com/skyrim/mods/27321) для сообщения об ошибках.'

  - name: 'WTF.esp'
    group: *latePatchGroup
    msg:
      - type: say
        content:
          - lang: en
            text: 'May need [TES5Edit](http://www.nexusmods.com/skyrim/mods/25859) cleaning. Several test generations of this SkyProc Patch included ITMs. A cleaning guide is available [here](https://www.creationkit.com/index.php?title=TES5Edit_Cleaning_Guide_-_TES5Edit).'
          - lang: de
            text: 'Möglicherweise muss mit [TES5Edit](http://www.nexusmods.com/skyrim/mods/25859) gesäubert werden. Mehrere Testgenerationen dieses SkyProc-Patches enthielten ITMs. Eine Säuberungsanleitung finden Sie [hier](https://www.creationkit.com/index.php?title=TES5Edit_Cleaning_Guide_-_TES5Edit).'
          - lang: es
            text: 'May need [TES5Edit](http://www.nexusmods.com/skyrim/mods/25859) cleaning. Several test generations of this SkyProc Patch included ITMs. Puedes encontra una guía [aquí](https://www.creationkit.com/index.php?title=TES5Edit_Cleaning_Guide_-_TES5Edit).'
          - lang: ja
            text: 'May need [TES5Edit]([TES5Edit](http://www.nexusmods.com/skyrim/mods/25859)でクリーニングする必要があるかもしれません。このSkyProc Patchでテスト生成するとITMが紛れこむ場合があります。クリーニングの手順は[こちら](https://www.creationkit.com/index.php?title=TES5Edit_Cleaning_Guide_-_TES5Edit)です。'
          - lang: ru
            text: 'Может понадобиться очистка в [TES5Edit](http://www.nexusmods.com/skyrim/mods/25859). Некоторые сгенерированные тестовые записи этого SkyProc патча являются ITM записями. Инструкция по очистке доступна [здесь](https://www.creationkit.com/index.php?title=TES5Edit_Cleaning_Guide_-_TES5Edit).'

  - name: 'XFLMain.esm'
    inc:
      - 'BDO.esp'
      - 'UFO - Ultimate Follower Overhaul.esp'
    msg:
      - type: say
        content:
          - lang: en
            text: 'For safe mod uninstall see special uninstallation instructions on [mod page](http://www.nexusmods.com/skyrim/mods/12933).'
          - lang: de
            text: 'Für eine sichere Deinstallation der Mod lesen Sie bitte die Deinstallationsanleitung auf der [Mod Seite](http://www.nexusmods.com/skyrim/mods/12933).'
          - lang: es
            text: 'Para desisntalar de manera segura por favor ve la [página del mod](http://www.nexusmods.com/skyrim/mods/12933).'
          - lang: ja
            text: '安全にアンインストールするには[Modのページ](http://www.nexusmods.com/skyrim/mods/12933)に記載されている専用のアンインストール手順を確認してください。'
          - lang: ru
            text: 'Для безопасного удаления мода смотрите инструкции по удалению на [странице мода](http://www.nexusmods.com/skyrim/mods/12933) или на [странице локализации](http://modgames.net/load/247-1-0-10650).'
```

</details>

**Comment by pStyl3** [comment]
The Witcher 2 plugins can be found [here](https://www.nexusmods.com/skyrim/mods/22054).

**Comment by sibir-ine** [comment]
The Unofficial Skyrim Patch hasn't been available for a long time, so it's probably safe to remove it entirely.

Sounds of Skyrim The Dungeons, The Wilds, & Civilization can be found [here](https://www.nexusmods.com/skyrim/mods/8601/), [here](https://www.nexusmods.com/skyrim/mods/10886/), & [here](https://www.nexusmods.com/skyrim/mods/20193/), respectively.

**Comment by pStyl3** [comment]
Regarding the Unofficial Skyrim Patch, I believe that USKP was a normal `.esp` before USLEEP came to be, so `Unofficial Skyrim Patch.esp`. I'm not sure where the `.esm` variant that is mentioned in the masterlist comes into the picture. Was it some version that was temporarily uploaded to USKP's modpage, or was it some plugin from a third party, unrelated to the UP team? What's your take regarding this @Arthmoor ?

**Comment by Arthmoor** [comment]
We have never issued an unofficial patch with the .esm extension. The ones for Skyrim were ESM flagged .esp files, so if there are .esm ones out there, they're not from us and I'd recommend against people using those because they may have been altered beyond that.

**Comment by pStyl3** [comment]
Thanks for clarifying. I'd guess, that there were some people recommending changing USKP's extension to `esm`, which in return resulted in this masterlist message not to do so and rather using USLEEP. In that sense it would probably still make sense to warn people if they use the `esm` plugin.

That being said, I can't find a discussion for this, nor a place where `Unofficial Skyrim Patch.esm` was uploaded. If it was once a thing, it's importance should be negligible at this point, if not non existent, and the number of people using USLEEP or (the original) USKP (or SSE with USEEP) should be way bigger. As such, we likely should be fine removing this masterlist entry after all.

**Comment by Arthmoor** [comment]
I have no idea where it would have been uploaded. I've never seen it myself, just occasionally ran into people who would mention it as a .esm file. It's been 6 years since the USLEEP integration, and it was never named as a .esm file either.

**Comment by pStyl3** [comment]
```yaml
SoS - The Dungeons.esp
SoS - The Wilds.esp
SoS - Civilization.esp

    msg:
      - type: say
        content:
          - lang: en
            text: 'Do not use the Crash Fix ESP for this mod. It is flagged as a master and is only meant as a last resort for fixing crashes.'
          - lang: de
            text: 'Verwenden Sie für diesen Mod nicht die Crash Fix ESP. Es ist als Master gekennzeichnet und nur als letzter Ausweg zur Behebung von Abstürzen gedacht.'
          - lang: ja
            text: 'このMod用のCrash Fix ESPは使用しないでください。マスターファイルとしてのフラグが立ちます。クラッシュを修正するための最後の手段としてください。'
          - lang: ru
            text: 'Не используйте Crash Fix ESP для этого мода. Он помечен как мастерфайл и предназначен только как последнее средство для исправления вылетов.'
```
The above mentioned `Crash Fix ESP` doesn't seem to be uploaded on the mod pages mentioned above by @sibir-ine.
But they are part of German translations for those mods:
- [Sounds of Skyrim - The Dungeons (Die Höhlen und Ruinen) 1.23](https://www.elderscrollsportal.de/ressourcen/sounds-of-skyrim-the-dungeons-die-hoehlen-und-ruinen.2448/)
- [Sounds of Skyrim - The Wilds (Die Wildnis) 1.13](https://www.elderscrollsportal.de/ressourcen/sounds-of-skyrim-the-wilds-die-wildnis.2447/)
- [Sounds of Skyrim - Civilization (Zivilisation) 1.02](https://www.elderscrollsportal.de/ressourcen/sounds-of-skyrim-civilization-zivilisation.2566/)

In their Readme they write:
> Sollten sich Abstürze häufen, benutze die ESP im Verzeichnis "03 Civilization Crash Fix".

That translates to:
> If crashes become more frequent, use the ESP in the "03 Civilization Crash Fix" directory. 

Since this message is currently without condition, we could add one for each plugin, with checksum & CRC.

Running xEdit's QAC on them gives these results:
```yaml
  - name: 'SoS - Civilization.esp'
    clean:
      - crc: 0x9B764A20
        util: 'TES5Edit v4.0.3g'

  - name: 'SoS - The Dungeons.esp'
    dirty:
      - <<: *quickClean
        crc: 0x8E705995
        util: '[TES5Edit v4.0.3g](https://www.nexusmods.com/skyrim/mods/25859)'
        itm: 3
    clean:
      - crc: 0x6ADB6D75
        util: 'TES5Edit v4.0.3g'

  - name: 'SoS - The Wilds.esp'
    dirty:
      - <<: *quickClean
        crc: 0x7989CAFE
        util: '[TES5Edit v4.0.3g](https://www.nexusmods.com/skyrim/mods/25859)'
        itm: 17
    clean:
      - crc: 0x7D64DFBB
        util: 'TES5Edit v4.0.3g'
```

**Comment by pStyl3** [comment]
`FastTravelAlle.esp` can be downloaded [here](https://www.nexusmods.com/skyrim/mods/27321) and [here](https://www.nexusmods.com/skyrim/mods/80392).

[Fasttravelfromanywhere.esp](https://www.nexusmods.com/skyrim/mods/8206)

`WTF.esp` comes from [this modpage](https://www.nexusmods.com/skyrim/mods/28761), though it's a plugin that only get's generated upon installation.

[The Spice of Life.esm](https://www.nexusmods.com/skyrim/mods/16668) **NSFW**

[XFLMain.esm](https://www.nexusmods.com/skyrim/mods/12933) - see old v3 versions

**Comment by pStyl3** [comment]
[Volrath - Dragonreach - Brazier fix - 1.1.esp](https://www.nexusmods.com/skyrim/mods/19371)

[WoodChoppingExtended.esp](https://www.nexusmods.com/skyrim/mods/16761)

[CK's Improved Windhelm.esp](https://www.nexusmods.com/skyrim/mods/15796)

I can't find `increaseddifficulty.esp`, so I'll assume it got deleted and as such, we also should delete the entry from the masterlist.
